src/components/Table/Table.js

```diff
@@ -55,7 +55,7 @@ const Table = ({
 
   useEffect(() => {
     const calculatePanelHeight = () => {
-      if (tableHeadRef && tableContentRef && tablePanelRef.current) {
+      if (tableHeadRef && tableContentRef && tablePanelRef.current && tableHeadRef.current) {
```

Comment:
can we have it like this?
```suggestion
      if (tableHeadRef?.current && tableContentRef?.current && tablePanelRef?.current) {
```